src/libs/protocols/protos/confidential_data_hub.proto

```diff
@@ -28,6 +28,24 @@ message SecureMountResponse {
     string mount_path = 1;
 }
 
+message ImagePullRequest {
+    // - `image_url`: The reference of the image to pull
+    string image_url = 1;
+
+    // - `bundle_path`: The path to store the bundle. After the RPC is
+    // called, Confidential Data Hub will put a `config.json` file
```

Comment:
What's the `config.json` file and why does the agent need it, given that it receives its OCI config.json from the runtime? If it's unused by the agent, we should not require implementations to create it.

Reply:
It's OCI config.json. It's the RPC result of image pulling, converted from [image configuration](https://github.com/opencontainers/image-spec/blob/main/config.md).

The [conversion document](https://github.com/opencontainers/image-spec/blob/main/conversion.md) says
'''
The "default generated runtime configuration" MAY be overridden or combined with externally provided inputs from the caller. In addition, a converter MAY have its own implementation-defined defaults and extensions which MAY be combined with the "default generated runtime configuration".
'''

So kata-agent can ignore it if do not use it.

Reply:
I understand that this is an implementation detail of image-rs, but, if we don't use it in the agent, why not remove it from the specification?

Reply:
Oh, I'm not sure if the RPC Proto definitions on both sides are compatible if the return values ​​are different. Let me add a patch to try it out.